app/models/instance_user.rb

```diff
@@ -1,8 +1,13 @@
 class InstanceUser < ActiveRecord::Base
+  include InstanceUserSearchConcern
   acts_as_tenant :instance, inverse_of: :instance_users
 
   enum role: { normal: 0, instructor: 1, administrator: 2 }
   belongs_to :user, inverse_of: :instance_users
 
   scope :ordered_by_username, -> { joins(:user).merge(User.order(name: :asc)) }
+
+  def self.search_and_ordered_by_username(keyword)
```

Comment:
@lowjoel Have a small issue here while combining `.search` and `.ordered_by_username`, the field in `order_by` must appear in `group_by`, so I added this method here to resolve it.